pkg/services/authn/authnimpl/sync/user_sync.go

```diff
@@ -282,11 +287,11 @@ func (s *UserSync) createUser(ctx context.Context, id *authn.Identity) (*user.Us
 	}
 
 	usr, errCreateUser := s.userService.Create(ctx, &user.CreateUserCommand{
-		Login:        id.Login,
-		Email:        id.Email,
-		Name:         id.Name,
-		IsAdmin:      isAdmin,
-		SkipOrgSetup: len(id.OrgRoles) > 0,
+		Login:   id.Login,
+		Email:   id.Email,
+		Name:    id.Name,
+		IsAdmin: isAdmin,
+		OrgID:   id.OrgID,
```

Comment:
I'm guessing it's because we assume the user orgs will be handled in https://github.com/grafana/grafana/blob/63cd5a5625c22dcb5603b10d54228363bcaa70a6/pkg/services/authn/authnimpl/sync/org_sync.go ?

Can't think right now of side effects of this

Reply:
Me neither. But I think we need to set `SkipOrgSetup: len(id.OrgRoles) > 0`